splunk_add_on_ucc_framework/dashboard.py

```diff
@@ -117,17 +119,30 @@
     '| rename event_account as \\"Account\\", events as \\"Number of events\\", '
     'sparkevent as \\"Event trendline\\"'
 )
+
 table_input_query = (
-    "index = _internal source=*{addon_name}* action=events_ingested "
+    '| rest splunk_server=local /services/data/inputs/all | where $eai:acl.app$ = \\"{addon_name}\\" '
+    '| eval Active=if(disabled=1, \\"no\\", \\"yes\\") | table title, Active '
```

Comment:
Do we get disabled as `1 and 0` or `true and false` or `T and F` as well?

Reply:
hmm I found disabled map only to `1 and 0` but since it is boolean datatype let;s handle `true and false` as well.